### ISSUE TITLE
NaN values are not allowed in Argo NetCDF variables

### ISSUE THREAD

**Issue by tcarval** [euroargodev/ArgoNetCDF#15]
NaN values are not allowed in Argo NetCDF variables.
The file format checker should report an error on NaN value in any variable.

**Comment by jegilson** [comment]
I agree with implementation of a general check to flag all netCDF with NaN.

**Comment by MarkIggy** [comment]
NaN checks are currently in place for all <PARAM>-related variables (physical parameters) in both profile and trajectory files.

NaNs are only an issue for floating-point data types - float and double.  Adding NaN checks to any of the variables that are already checked by the FileChecker would be relatively straightforward.  Adding NaN checks for every float/double variable in all of the file types will probably 

I will have to review the file specfications to see how extensive the changes will be for adding NaN checks for every float/double variable.

**Comment by delphinedobler** [comment]
First item has been corrected since release 2.9.0
Extract from https://github.com/OneArgo/ArgoFormatChecker/pull/30
fix : NaN and Inf values in PARAM, PARAM_ADJUSTED and PARAM_ADJUSTED_ERROR' are now correctly detected and raise an error if present.

Second part relevance remains to be assessed.

**Comment by delphinedobler** [comment]
JPR also noted some "-Inf" values: 
- [ ] verify this with respect to what was coded in the fix.
